src/app/fsat/explore-opportunities/explore-opportunities-form/system-data-form/system-data-form.component.html

```diff
@@ -5,28 +5,16 @@
         Reduce System Flow Rate
     </label>
 </div>
-<li *ngIf="showFlowRate && !isVFD">
-    <form [formGroup]="baselineForm">
-        <div class="form-group" [ngClass]="{'invalid': baselineForm.controls.flowRate.invalid}">
-            <label for="baselineFlowRate">Baseline Flow Rate</label>
-            <div class="input-group">
-                <input name="baselineFlowRate" type="number" step="any" id="baselineFlowRate" class="form-control"
-                    (focus)="focusField('flowRate')" (input)="calculate()" formControlName="flowRate" />
-                <span class="input-group-addon units" [innerHTML]="settings.fanFlowRate | settingsLabel"></span>
-                <!--<span class="input-group-addon units">{{settings.fanFlowRate}}</span>-->
-            </div>
-            <span class="alert-danger pull-right small"
-                *ngIf="baselineForm.controls.flowRate.invalid && !baselineForm.controls.flowRate.pristine">
-                <span *ngIf="baselineForm.controls.flowRate.errors.required">Value Required</span>
-                <span *ngIf="baselineForm.controls.flowRate.errors.min">Value can't be less than
-                    {{baselineForm.controls.flowRate.errors.min.min}} <span
-                        [innerHTML]="settings.fanFlowRate | settingsLabel"></span></span>
-            </span>
+<li *ngIf="showFlowRate && !isVFD" class="explore-opps-item">
+    <form [formGroup]="baselineForm" class="explore-opps-form">
+        <div class="form-group explore-opps-formgroup" [ngClass]="{'invalid': baselineForm.controls.flowRate.invalid}">
+            <label for="baselineFlowRate" class="baseline-value-label">Baseline Flow Rate</label>
+            <div class="text-muted baseline-value-input">{{ baselineForm.controls.flowRate.value }} {{settings.fanFlowRate || settingsLabel }}</div>
```

Comment:
Let's add the number pipe onto this.

Additionally, pipes only use one | so it should be `| settingsLabel` we also use [innerHTML] for the settingsLabel because it has hex codes for super and subscripts

Reply:
Apply in all places using pipes